src/server/pfs/server/paths.go

```diff
@@ -59,10 +57,11 @@ func cleanPath(x string) string {
 	return "/" + strings.Trim(x, "/")
 }
 
-func checkFilePath(path string) error {
-	path = filepath.Clean(path)
-	if strings.HasPrefix(path, "../") {
-		return errors.Errorf("path (%s) invalid: traverses above root", path)
-	}
-	return nil
-}
+// TODO: Readd?
```

Comment:
What happened here?

Reply:
The linter complained about it being unused. I think it probably makes sense to just throw it away for now similar to the copy file validation in the validation server (like you mentioned below).